### DISCUSSION TITLE
Deduplication Question

### DISCUSSION THREAD

**Discussion by mspencer92** [moj-analytical-services/splink#213] [Q&A] [answered]
Hello everyone, after I call linker.get_scored_comparisons() what would be the best way to group the duplicates together so I can remove the duplicates?

**Comment by mamonu** [comment, marked as answer]
One of the ways of achieving this in the pyspark domain is by using the connected components functionality available from the graphframes library. This way you will be able to group the (possible) duplicates together

Once you have this then perhaps you could keep the highest score (or have any other business logic rule)

an example of such use is here.
https://github.com/moj-analytical-services/splink_demos/blob/master/deduplication_detailed_example.ipynb

in particular have a look at  `Step 7: Use graphframes to resolve links into groups`

**Reply by aalexandersson** [reply]
The link [https://github.com/moj-analytical-services/splink_demos/blob/master/deduplication_detailed_example.ipynb](https://github.com/moj-analytical-services/splink_demos/blob/master/deduplication_detailed_example.ipynb) is no longer available. Did you delete it or merely move it somewhere else?
